src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java

```diff
@@ -180,8 +180,13 @@ private void addAssignmentSummaryToPage(ModelAndView page, int totalPresentDays,
             double totalExcusedPoints = Math.round(excusedDaysTimesMultiplier * assignmentPoints);
 
             double sumStudentsPoints = totalPresentPoints + totalTardyPoints + totalAbsentPoints + totalExcusedPoints;
-
-            double studentFinalGrade = Math.round((sumStudentsPoints / totalDays) * 100.0) / 100.0;
+            double studentFinalGrade;
+            if (totalDays == 0){
+                studentFinalGrade = Math.round((sumStudentsPoints / 1) * 100.0) / 100.0;
```

Comment:
(sumStudentsPoints/1) would always be equal to sumStudentsPoints. I think the /1 can be removed.